src/model/routes/modelRouter.ts

```diff
@@ -6,6 +6,7 @@ const modelRouterFactory: FactoryFunction<Router> = (dependencyContainer) => {
   const router = Router();
   const controller = dependencyContainer.resolve(ModelController);
 
+  router.post('/:identifier', controller.deleteModel);
```

Comment:
This is supposed to be delete
```suggestion
  router.delete('/:identifier', controller.deleteModel);
```

Reply:
this is post request for deleting model.

Reply:
post is usually used to create something. You want to delete.

Reply:
im creating request for delete...

Reply:
When you send a DELETE request, you are indicating that the resource identified by the URI should be deleted. In your case, you want to delete the model even if you are not deleting it directly. Using the DELETE method would be more semantically appropriate.

Reply:
ok, i will change it to delete